site/src/style/variables.less

```diff
@@ -18,3 +18,12 @@
 @jumbotron-color:       white;
 
 @section-vertical-padding: 30px;
+
```

Comment:
I think that's a reasonable progression and it's [standard enough](http://spencermortensen.com/articles/typographic-scale/). Maybe we could boost the base-font-size to make them more visually different?